tool/tctl/common/plugin/awsic_test.go

```diff
@@ -0,0 +1,196 @@
+// Teleport
+// Copyright (C) 2025 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package plugin
+
+import (
+	"math/rand/v2"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/tool/common"
+)
+
+func TestAWSICUserFilters(t *testing.T) {
+	t.Run("parse", func(t *testing.T) {
```

Comment:
Nit: I don't see the point in using subtests here since the subtests don't share any common setup logic. 

Using subtests just to add a layer of nesting as is common with various BDD frameworks doesn't feel very idiomatic to me.

Reply:
Flattened